src/components/Match/MatchStory.jsx

```diff
@@ -152,6 +140,29 @@ class StoryEvent {
   }
 }
 
+class IntroEvent extends StoryEvent {
+  constructor(match) {
+    super(-90);
+    this.game_mode = match.game_mode;
+    this.region = match.region;
+    this.date = new Date(match.start_time * 1000);
+  }
+  get localizedRegion() {
```

Comment:
If we want to use a nicer looking string for region, you may want to consider doing this in odota/dotaconstants instead, then it would show up the same way e.g. on match pages.